sandbox/auth/session/token.go

```diff
@@ -2,16 +2,40 @@ package session
 
 import (
 	"github.com/go-jose/go-jose/v3"
+	"github.com/go-jose/go-jose/v3/jwt"
 	"golang.org/x/oauth2"
 )
 
+// Tokens can generally be ID or access. It's a bit weird to have a struct
+// named "Token" that only contains an ID token. An alternative is to just rename
+// this struct to IDToken.
 type Token struct {
-	oauth2.Token // Embed and oauth2 token
```

Comment:
I don't think we should remove this (although I'm open to hearing why you think we should). The idea was to "extend" the standard Oauth token.

This embedded struct does a couple of things:
1. Ensures all the appropriate oauth2.Token fields get serialized when we convert to JSON (for example, the Access Token is getting serialized because of this field, but no longer would be once you remove)

2. It lets us pass our token to places that need an `oauth2` token without a conversion.

3. It gives us access to methods we don't have to re-implement. For example, I was going to use `.Valid()` (https://pkg.go.dev/golang.org/x/oauth2#Token.Valid) to address this bug: "New auth lib returns ID token even if expired".

Reply:
I'm a bit confused because in line 12 in sandbox/auth/session/oauth.go you are passing the ID token into this field, so we essentially have it twice.

The `AccessToken` field inside is a bit unfortunatel becase it doesn't look like anything in this token is specifically access vs ID.

We can keep this, but then we should remove the raw string IDtoken. Or if this is meant to be the access token then we should rename the struct to tokenSet and the fields accordingly.